helm/robusta/templates/runner.yaml

```diff
@@ -131,4 +137,13 @@ spec:
   selector:
     matchLabels:
       app: {{ .Release.Name }}-runner
+{{ end }}
+{{ if .Values.runner.sentry_dsn }}
```

Comment:
I think the manifest separator is missing here '---'
It works without it, but it's easier to read